github-actions/src/main/scala/org/typelevel/sbt/gha/GenerativeKeys.scala

```diff
@@ -51,6 +51,7 @@ trait GenerativeKeys {
     settingKey[Seq[MatrixExclude]]("A list of matrix exclusions (default: [])")
   lazy val githubWorkflowBuildRunsOnExtraLabels = settingKey[Seq[String]](
     "A list of additional labels to append to each run of the matrix executions")
+  @annotation.nowarn("msg=possible missing interpolator")
   lazy val githubWorkflowBuildConcurrency = settingKey[Option[JobConcurrency]](
     "Use concurrency to ensure that only a single build job within the same concurrency group will run at a time. (default: ci-${{ matrix.os }}-${{ matrix.java }}-${{ matrix.scala }}-${{ github.ref }})")
```

Comment:
Instead of the `nowarn` this is the trick we usually use, but I don't have a strong opinion about it.

```suggestion
  lazy val githubWorkflowBuildConcurrency = settingKey[Option[JobConcurrency]](
    s"Use concurrency to ensure that only a single build job within the same concurrency group will run at a time. (default: ci-$${{ matrix.os }}-$${{ matrix.java }}-$${{ matrix.scala }}-$${{ github.ref }})")
```